grazer/src/main/java/edu/asu/diging/grazer/core/domain/IFileImpl.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.diging.grazer.core.domain;
+
+import java.util.Date;
+import java.util.List;
+
+public interface IFileImpl {
+
+    int getId();
+    
+    String getLabel();
+
+    void setLabel(String label);
+
+    String getDescription();
+
+    void setDescription(String description);
+    
+    List<byte[]> getData();
+
+    void setData(List<byte[]> data);
+    
+    List<String> getfileNames();
```

Comment:
- needs to use camel case
- why is it a list of filenames? IFileImpl sounds like it represents one file.

Reply:
now I get it. IFile is a bad name if it represents one transformation. Also, instead of using a list of filesname and a list of corresponding byte arrays, it would be much less error prone to have a list of file objects (whatever you want to call the class) that map filename to byte array.